modules/ROOT/pages/mongodb/mongodb-connector-6-0-studio.adoc

```diff
@@ -0,0 +1,100 @@
+= MongoDB Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+To configure a connector in Anypoint Studio follow this process:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+== Install the Connector Using Exchange
+
+. In Studio, create a Mule project.
+. Click the Exchange *(X)* icon in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `MongoDB`.
+. Select the MongoDB Connector and click *Add to project*.
+. Follow the prompts to install the connector.
+
+== Install the Connector in Studio
+
+. In Studio, create a Mule project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the name of the connector in the search field.
+. Click the MongoDB Connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source for the Connector
+
+Configure an input source for the connector such as the Object Listener operation, HTTP Listener, or Scheduler. The Object Listener operation invoke your application when MongoDB has a new object in a specified collection.
+
+To configure an input source, drag it to the *Source* area of the Studio canvas.
+
+== Configure the Connector
+
+. On the *Global Elements* tab, click *Create*.
+. Expand *Connector Configuration* and search for `MongoDB config`.
+. Click *OK*.
+. Enter a value for the following properties:
+* *Servers (host:port)*
++
+Comma-separated list of host:port instances. The default value is `localhost:27017`.
++
+* *Database*
++
+Database that sends and retrieves data from the MongoDB instance
++
+* *Username*
++
+Username to access the MongoDB instance
++
+* *Password*
++
+Corresponding password
++
+. In the *Connection* section, select the *Advanced* tab to adjust the values for the *Connection timeout*, *Socket timeout*, and *Maximum connection idle time* field.
++
+The default value for the *Connect timeout* field is 30000 milliseconds (30 seconds). Any connection requests that take longer than 30 seconds to send and receive throw an exception.
++
+If you set the *Connection timeout* field to 0, MongoDB Connector waits indefinitely until a request is successfully sent and a response successfully received.
++
+. In the *Collection metadata samples* field, optionally specify existing collections from which to take the metadata structure.
+
+It is recommended that you use the MongoDB configuration with Spring property placeholders for the *Connection* field values when you plan to deploy your application to CloudHub or to a Mule runtime. However, you can hard code your connection credentials in the global element properties if you are in the development stage and want to speed up the development process.
+
+=== Global Configuration with SSL (Optional)
```

Comment:
Because this is a solitary subtopic of Configure the Connector, and very short, I wonder if we should not just merge it into its container topic. Or should it be promoted to be a peer of Configure the Connector?